src/test/java/jerry/codecamp/DogTest.java

```diff
@@ -126,12 +126,64 @@ public void timePassesForDog_increasesTiredLevel(){
     }
 
     @Test
-    public void timePassesForDog_increasesThirstLevel(){
+    public void timePassesForDog_increasesThirstLevel() {
         dog.timePassesForDog();
         assertEquals(5, dog.getThirstLevel());
 
         dog.timePassesForDog();
         assertEquals(10, dog.getThirstLevel());
     }
 
+    @Test
+    public void giveTheDogASnack_increasesHungerLevel() {
+        dog.giveTheDogASnack();
+        assertEquals(- 10, dog.getHungerLevel());
+
+        dog.giveTheDogASnack();
+        assertEquals(-20, dog.getHungerLevel());
+
+    }
```

Comment:
Can a dog level go below 0?

Reply:
I notice I should the method name to decreases as well